### PR TITLE
Add CentOS/RHEL 8 support

### PR DESCRIPTION
**Pull request by bahner**
SImple PR to make module work on rhel8 et al.

### CODE REVIEW

**Comment by ekohl** [comment]
I'd like to really support it. That means it also should be mentioned in `metadata.json` and let Travis run acceptance tests on it.

**Comment by bahner** [comment]
Okey dokey. Will look at ut during the weekend.

**Review by bastelfreak — APPROVED** [review]

**Comment by ekohl** [comment]
We have acceptance tests in this module. I'd like to run them on CentOS 8 as well. We enable those by adding it here:
https://github.com/voxpupuli/puppet-gluster/blob/ae43a3420177a0adb44669abee3332adcb828447/.sync.yml#L4

Then our modulesync generates lines like:
https://github.com/voxpupuli/puppet-gluster/blob/ae43a3420177a0adb44669abee3332adcb828447/.travis.yml#L26-L33

In practice I found it easier for things like this to manually copy them than to run modulesync for the single module in your PR.

**Comment by bahner** [comment]
Well pardon my french, but I've never used travis before ;-) It's fun to learn, though. But I might need a few commits. We apologize for the inconvenience.

**Comment by bahner** [comment]
So, this fails for pulling redhat. It seems that redhat7 and redhat8 are not avaiable. Should I just use centos and treat them as qualifying redhat?

**Comment by ekohl** [comment]
Don't worry - we all had to learn it some time.

**Comment by bahner** [comment]
So the repository  PowerTools needs to be enabled on Centos8. I need to think about how to make sure that is enabled in a good way. I prolly need to move a config of this up closer to the user. Suggestions are welcome for how to solve this.

File: manifests/params.pp [hunk 1]
```diff
@@ -24,12 +24,11 @@
...
 
       $server_package = $facts['os']['release']['major'] ? {
-        # RHEL 6 and 7 provide Gluster packages natively
-        /(6|7)/ => 'glusterfs-server',
+        /(6|7|8)/ => 'glusterfs-server',
         default => false
       }
       $client_package = $facts['os']['release']['major'] ? {
-        /(6|7)/ => 'glusterfs-fuse',
+        /(6|7|8)/ => 'glusterfs-fuse',
```
Comment (bastelfreak) [inline manifests/params.pp:31]: we could also make this more readable my dropping the selector here and jus set `$client_package` to `glusterfs-fuse`. same for the server_package.

**Comment by ekohl** [comment]
Perhaps the release is incorrect? Looks like the release is now 7.3 but the latest is Gluster 8. Gluster 8 does include `python3-pyxattr`. It could also be a bug in the Gluster 7 packaging in the SIG. Have you talked to upstream?

**Comment by ekohl** [comment]
I've submitted https://github.com/voxpupuli/puppet-gluster/pull/226 with the simplification in params.pp. Will see if that passes.

**Pull request closed without merging**